src/geocat/viz/util.py

```diff
@@ -1118,6 +1243,72 @@ def plotELabels(da: xr.DataArray,
     - `NCL_sat_2.py <https://geocat-examples.readthedocs.io/en/latest/gallery/MapProjections/NCL_sat_2.html?highlight=plotELabels>`_
     """
 
+    warnings.warn(
+        "This function is deprecated. Please use `plot_extrema_labels` instead."
+    )
+
+    return plot_extrema_labels(da, transform, proj, clabel_locations, label,
+                               fontsize, whitebbox, horizontal)
+
+
+def plot_extrema_labels(da: xr.DataArray,
+                        transform: cartopy.crs.CRS,
+                        proj: cartopy.crs.CRS,
+                        label_locations: list = [],
+                        label: str = 'L',
+                        fontsize: int = 22,
+                        whitebbox: bool = False,
+                        horizontal: bool = True) -> list:
+    """Utility function to plot contour labels.
+
+    High/Low contour labels will be plotted using text boxes for more accurate label values
+    and placement.
+    This function is exemplified in the python version of https://www.ncl.ucar.edu/Applications/Images/sat_1_lg.png
```

Comment:
It might be worth making this into a pretty link while you're moving stuff around anyway